packages/upswyng-server/src/components/Autocomplete.svelte

```diff
@@ -0,0 +1,72 @@
+<script context="module">
+  export const key = {};
+</script>
+
+<script>
+  import { afterUpdate, onDestroy, setContext } from "svelte";
```

Comment:
think you've got some unused imports here

Reply:
ahh, good call. [updated](https://github.com/CodeForBoulder/upswyng/pull/332/commits/cbedf4b2e7e6ce046ed2369ee71499a99df29885)